nixos/modules/services/misc/tautulli.nix

```diff
@@ -74,6 +78,10 @@ in
       };
     };
 
+    networking.firewall = mkIf cfg.openFirewall {
```

Comment:
Minor nit, but this can be changed to simply `networking.firewall.allowedTCPPorts = mkIf ...`